packages/searchbase/src/types.js

```diff
@@ -0,0 +1,51 @@
+type Suggestion = {
```

Comment:
What's the need of creating a separate file for types?
Types are more readable when they exist in the same file.

Reply:
let's say we introduce `i18n`, it is always better to have all the strings in one place. Also, it becomes easier to search if all the `types` of the project lies at one place